blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_18.py

```diff
@@ -35,3 +36,13 @@ def _evaluate(
         """Evaluates the formula, for more information see the __init__ method."""
         raise_if_negative(l_pt=l_pt)
         return 1.2 * l_pt
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.18."""
+        return LatexFormula(
+            return_symbol=r"l_{pt1}",
```

Comment:
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/1618a9c7-6814-4e0b-89ea-49fa78dbdb1c)
```suggestion
            return_symbol=r"l_{pt2}",
```